kv/kv.go

```diff
@@ -343,6 +343,7 @@ const (
 )
 
 // StoreType represents the type of a store.
+// Type enums should sync with store/tikv.StoreType.
```

Comment:
Why not unify these two `StoreType`?

Reply:
good point.

Reply:
@andylokandy I discover that the 2 `storeType`s are actually not strict related. There is no need to keep them synced. (I tried to change `tikv.StoreType` to type string, it can still compile). So I removed the comments that declare they should be sync.